spec/requests/health_care_applications_request_spec.rb

```diff
@@ -17,17 +17,29 @@
 
     before do
       sign_in_as(current_user)
+      stub_mpi
     end
 
     it 'returns the users rating info' do
-      VCR.use_cassette('bgs/service/find_rating_data', VCR::MATCH_EVERYTHING) do
+      VCR.use_cassette('bgs/service/find_rating_data') do
         get(rating_info_v0_health_care_applications_path)
       end
 
       expect(JSON.parse(response.body)['data']['attributes']).to eq(
         { 'user_percent_of_disability' => 100 }
       )
     end
+
+    it 'errors if user is not loa3' do
+      VCR.use_cassette('bgs/service/find_rating_data') do
```

Comment:
you don't need to use VCR here, it's supposed to block the user from the API and it shouldn't be making a call to BGS